packages/oasis-actions/src/helpers/calculations/Position.ts

```diff
@@ -143,6 +145,22 @@ export class Position implements IPosition {
     return new RiskRatio(ltv, RiskRatio.TYPE.LTV)
   }
 
+  public get maxDebtToBorrow() {
+    const maxLoanToValue = this.category.maxLoanToValue
+    return this.collateral.normalisedAmount
+      .times(this._oraclePriceForCollateralDebtExchangeRate)
+      .times(maxLoanToValue)
+      .minus(this.debt.normalisedAmount)
+  }
+
+  public get maxCollateralToWithdraw() {
+    const maxLoanToValue = this.category.maxLoanToValue
+    const minimumCollateral = this.debt.normalisedAmount.div(
+      maxLoanToValue.times(this._oraclePriceForCollateralDebtExchangeRate),
+    )
+    return this.collateral.amount.minus(minimumCollateral)
+  }
+
```

Comment:
In general don't you think this should be a utility function? `maxDebtToBorrow(position): BigNumber`?